app/models/order.rb

```diff
@@ -15,17 +15,7 @@ def self.count_by_status
     Order.group(:status).count
   end
 
-  def self.sorted_orders(sort_by='all')
-    if sort_by == 'ordered'
-      Order.ordered
-    elsif sort_by == 'paid'
-      Order.paid
-    elsif sort_by == 'cancelled'
-      Order.cancelled
-    elsif sort_by == 'completed'
-      Order.completed
-    else
-      Order.all
-    end
```

Comment:
this is sheer proof that delirium does set in to the immortals.  skynet has won......Nice refactor